sdk/test/Services/SecurityToken/UnitTests/Custom/StsRegionalFlagTests.cs

```diff
@@ -13,77 +13,23 @@
   * permissions and limitations under the License.
  */
 using Amazon.SecurityToken;
-using Amazon.SecurityToken.Model;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
 using System;
-using System.Reflection;
-using System.Text;
-using System.Globalization;
-using Amazon.Runtime.CredentialManagement;
 using Amazon.Runtime;
-using AWSSDK_DotNet.CommonTest.Utils;
 
 namespace AWSSDK.UnitTests
 {
     [TestClass]
     public class StsRegionalFlagTests
     {
-        private static readonly string ProfileText = @"[default]
-                                                        region=us-west-2
-                                                        aws_access_key_id=default_aws_access_key_id
-                                                        aws_secret_access_key=default_aws_secret_access_key
-                                                        sts_regional_endpoints=legacy
-                                                        [other]
-                                                        region=us-west-2
-                                                        aws_access_key_id=other_aws_access_key_id
-                                                        aws_secret_access_key=other_aws_secret_access_key
-                                                        sts_regional_endpoints=regional";
-       
-
-        private const string AwsProfileEnvironmentVariable = "AWS_PROFILE";
         private const string AwsStsRegionalEndpointsEnvironmentVariable = "AWS_STS_REGIONAL_ENDPOINTS";
 
-        [Ignore]
-        [DataTestMethod]
-        [DataRow("default", StsRegionalEndpointsValue.Legacy)]
-        [DataRow("other", StsRegionalEndpointsValue.Regional)]
-        [TestCategory("SecurityToken")]
-        /// Ensure that the STS Regional Flag is set according to the value
-        /// in the Shared Credentials File
-        public void TestCredentialsFileStsFlag(string profile, StsRegionalEndpointsValue expectedValue)
```

Comment:
Moved this test to `sdk/test/UnitTests/Custom/Runtime/Credentials/SharedCredentialsFileTest.cs` (which has access to the `SharedCredentialsFileTestFixture`).